pandas/core/tools/datetimes.py

```diff
@@ -2,8 +2,14 @@
 import numpy as np
 from collections import MutableMapping
 
-from pandas._libs import lib, tslib
+from pandas._libs import tslib
 from pandas._libs.tslibs.timezones import get_timezone
+from pandas._libs.tslibs import parsing
+from pandas._libs.tslibs.parsing import (  # noqa
+    parse_time_string,
+    _format_is_iso,
```

Comment:
can come back in future and de-privatize these in tslib.parsing (and change here)

Reply:
Just added to TODO list in #17652